### PR TITLE
fixes broken CI

### PR DESCRIPTION
**Pull request by ringabout**
ref https://github.com/nim-lang/packages/pull/2576

### CODE REVIEW

File: testament/important_packages.nim [hunk 1]
```diff
@@ -136,7 +136,7 @@ pkg "pylib"
 pkg "rbtree"
 pkg "react", "nimble example"
 pkg "regex", "nim c src/regex"
-pkg "result", "nim c -r result.nim"
+pkg "results", "nim c -r result.nim"
```
Comment (ringabout) [inline testament/important_packages.nim]:
```suggestion
pkg "results", "nim c -r results.nim"
```

**Comment by ringabout** [comment]
See also https://github.com/Yardanico/nimpylib/pull/24/files#r1177428189

**Comment by ghost** [comment]
@ringabout already fixed, sorry

**Comment by ringabout** [comment]
@Yardanico thanks for your quick response!

**Pull request merged**